cedar-policy-core/src/evaluator/err.rs

```diff
@@ -42,19 +41,20 @@ pub enum EvaluationError {
     #[error("cannot access attribute of unspecified entity: {0}")]
     UnspecifiedEntityAccess(SmolStr),
 
-    /// Tried to get this attribute of a (non-entity) record, but that record
+    /// Tried to get an attribute of a (non-entity) record, but that record
     /// didn't have that attribute
-    #[error("record does not have the required attribute: {0}")]
+    #[error("record does not have the attribute: {0}")]
     RecordAttrDoesNotExist(SmolStr),
 
-    /// Error thown by an operation on `Extensions`
-    /// (not to be confused with `ExtensionError`, which is an error thrown by
-    /// an individual extension function)
+    /// An error occurred when dispatching an extension function call. This
+    /// should not be confused with `ExtensionError`, which is an error thrown
```

Comment:
I would prefer another name like `ExtensionDispatchError` because it's too close to `ExtensionError`.

Reply:
I agree that `ExtensionError` vs `ExtensionsError` is confusing. Will push an update with new names later today.